static/app/utils/discover/fields.tsx

```diff
@@ -749,6 +749,7 @@ export const ERROR_ONLY_FIELDS: (FieldKey | SpanOpBreakdown)[] = [
   FieldKey.STACK_COLNO,
   FieldKey.STACK_LINENO,
   FieldKey.STACK_STACK_LEVEL,
+  FieldKey.EVENT_TYPE,
```

Comment:
Just to confirm, this is because we can have `default`, `csp`, etc as error types right? Good catch.